### DISCUSSION TITLE
Pretalx version on webUI

### DISCUSSION THREAD

**Discussion by Rufus125** [pretalx/pretalx#1588] [Q&A] [answered]
Hey!

I would like to know which version our pretalx instance is.

Is the version of pretalx visible somewhere? 
Or do I need to check the python package?

**Comment by rixx** [comment]
There's currently no way to see your pretalx version in the frontend. I'll add one to be included in the next release in the administrator dashboard!

**Reply by rixx** [reply, marked as answer]
Version information is included in the admin dashboard as of 7ecf37d.
